File: sdks/typescript/src/environments.ts
export function getServerUrlByKey(key: string): string {
    if (key.startsWith("gram_local_")) {
        return "http://localhost:8080";
    } else if (key.startsWith("gram_test_")) {
        return "https://dev.getgram.ai";
    } else if (key.startsWith("gram_live_")) {
        return "https://getgram.ai";
    } else {
        return "https://getgram.ai";
    }
}

File: sdks/python/src/gram_ai/environments.py
def get_server_url_by_key(key: str) -> str:
    if key.startswith("gram_local_"):
        return "http://localhost:8080"
    if key.startswith("gram_test_"):
        return "https://dev.getgram.ai"
    if key.startswith("gram_live_"):
        return "https://api.getgram.ai"
    return "https://api.getgram.ai"
